shell/platform/embedder/embedder.h

```diff
@@ -961,6 +961,8 @@ typedef struct {
   double scale;
   /// The rotation of the pan/zoom in radians, where 0.0 is the initial angle.
   double rotation;
+  /// The identifier of the view that received the pointer event.
+  int64_t view_id;
```

Comment:
Please introduce a type for the view identifiers. See this section in the design: https://docs.google.com/document/d/1kegGJ235y1778-vzyyv90MgSHmYQcROTupMuLtEDBq0/edit#heading=h.u7to6t120q4